README.adoc

```diff
@@ -63,7 +63,7 @@ sense. With SDKMAN!, the `~/.m2/mvnd.properties` file is typically not needed at
 
 [source,shell]
 ----
-$ brew install mvndaemon/homebrew-mvnd/mvnd
+$ brew install mvndaemon/homebrew-mvnd/mvnd@1
```

Comment:
This PR should target `mvnd-1.x` branch and not `master`.

Reply:
I'm targeting the README file that appears when you got to the repository. That would the master branch. Should the master branch have instructions that install the pre-release version?

Also, MVND is awesome.

Reply:
Well, maybe provide instructions for both then ? and add a comment to explain what that there are two branches.